code/modules/mob/living/simple_animal/hostile/megafauna/bubblegum.dm

```diff
@@ -135,12 +135,12 @@ Difficulty: Hard
 /mob/living/simple_animal/hostile/megafauna/bubblegum/Move()
 	if(charging)
 		new /obj/effect/temp_visual/decoy/fading(loc,src)
-		DestroySurroundings()
```

Comment:
this was being abused as a way to clear out mineral turfs(and other stuff) in the area around it, so some variant of the original behavior would be helpful, especially since the new proc always aims at where you actually are(so if you dodge bubblegum's charge it'll end up stopping halfway through rock/other stuff)

Reply:
i didnt test the megafauna so ill try to fix it so that they actually destroy their surroundings
it really needed to be a different proc anyway

Reply:
I mean, for this specific megafauna it was being used to destroy the area around it, and it needs some sort of similar effect so it can smash through walls, windows, and other relevant structures while charging.